### PR TITLE
Add h1, h2 to artist head for SEO

### PR DESCRIPTION
**Pull request by zephraph**
This fixes [DISCO-504](https://artsyproduct.atlassian.net/browse/DISCO-504).

Essentially just wraps the artist name in an h1 and the artist tombstone in an h2. 

Note that I don't necessarily consider this the best way to solve this problem. I feel like allowing the typography components to accept an `is` prop that can redefine the element in renders in would be the better way to go. In that case, we could change the underlying element whenever we wanted so it'd be a lot more robust. 

Unfortunately that's also a lot of work. Updating the types and not breaking react native are both things that I'd be concerned with there. Maybe a good item to pair with someone on later.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 6e1d7c00d23ed2be496299f88fa39cda926fa75b

https://deploy-preview-1608--artsy-reaction.netlify.com

**Comment by zephraph** [comment]
Before this is merged I need to test on some other browsers. I don't think unset has the best support.

**Comment by mzikherman** [comment]
LGTM!

File: src/Apps/Artist/Components/ArtistHeader.tsx [hunk 1]
```diff
@@ -6,9 +6,27 @@ import { ContextConsumer, Mediator } from "Artsy/SystemContext"
...
 import React, { Component, Fragment } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
 import { Carousel } from "Styleguide/Components"
 import { Media } from "Utils/Responsive"
 
+/**
+ * This H1 and H2 were added for SEO purposes
+ */
+const H1 = styled.h1`
```
Comment (oxaudo) [inline src/Apps/Artist/Components/ArtistHeader.tsx]: Should this leave somewhere more global? We will probably have to do the same on Artwork page and other pages once we start rebuilding them in responsive.

Comment (zephraph) [reply src/Apps/Artist/Components/ArtistHeader.tsx]: I think that's a great point.

**Comment by zephraph** [comment]
@mzikherman how does it look now?

**Pull request merged**